core/memory/working_memory.py

```diff
@@ -0,0 +1,96 @@
+import sqlite3
+import json
+
+class WorkingMemory:
+    def __init__(self, db_path="db.db"):
+        self.conn = sqlite3.connect(db_path)
+        self.create_tables()
+
+    def create_tables(self):
+        cursor = self.conn.cursor()
+        cursor.execute('''
+            CREATE TABLE IF NOT EXISTS MemoryModules (
```

Comment:
I think to give a meaningful review of this I'd have to go back over how memory modules are intended to work, but it seems like its just a KV store and given that this makes sense.